site/src/app/v2/trouver-un-club/components/disability-tag/DisabilityTag.tsx

```diff
@@ -0,0 +1,22 @@
+import Tag from '@codegouvfr/react-dsfr/Tag';
+import { Club } from 'types/Club';
+import styles from './styles.module.scss';
+
+interface Props {
+  club: Club;
+}
+export const DisabilityTag = ({ club }: Props) => {
+  const { a_accueil_handicap_mental, a_accueil_handicap_moteur } = club;
+
+  if (a_accueil_handicap_mental && a_accueil_handicap_moteur) {
+    return <Tag>Handicap moteur/mental</Tag>;
```

Comment:
Ce tag est cense avoir la couleur green-menthe-925 d'apres le figma

Reply:
dans la version ready for dev, ce n'est pas le cas. Je me suis basé sur ca (pas sur la maquette 'brouillon')